src/server/routes/__tests__/submissions.integration.spec.js

```diff
@@ -571,9 +574,21 @@ describe('submissions.integration', () => {
     })
   })
 
+  // TODO finish integration tests off
```

Comment:
Please create tickets rather than using TODO comments

Reply:
I forgot to remove this, I actually finished the integration tests off!